src/node_api.h

```diff
@@ -614,6 +614,36 @@ NAPI_EXTERN napi_status napi_run_script(napi_env env,
 NAPI_EXTERN napi_status napi_get_uv_event_loop(napi_env env,
                                                struct uv_loop_s** loop);
 
+// Calling into JS from other threads
+NAPI_EXTERN napi_status
+napi_create_threadsafe_function(napi_env env,
```

Comment:
Somehow threadsafe makes me think of something else but it might just be me.  I don't have great alternative suggestions though the best I've come up with is `napi_create_thread_callable_function()`